server/core/runtime/apply_step_runner_test.go

```diff
@@ -79,7 +79,7 @@ func TestRun_Success(t *testing.T) {
 func TestRun_AppliesCorrectProjectPlan(t *testing.T) {
 	// When running for a project, the planfile has a different name.
 	tmpDir := t.TempDir()
-	planPath := filepath.Join(tmpDir, "projectname-default.tfplan")
+	planPath := filepath.Join(tmpDir, "projectname::default.tfplan")
```

Comment:
Why use `::` ?

Reply:
Long story: https://github.com/runatlantis/atlantis/pull/2921#issuecomment-1379023019
Short story: `-` can be used by the users in the project and/or workspace, so using it as a separator could lead to problems, it was possible to use this before because the workspace name was part of the clone directory, so a `substring` function could be used to get the project name, since we don't have this anymore `::` was the safest separator I could find